packages/labs/cli/src/lib/generate/generate.ts

```diff
@@ -93,12 +110,18 @@ export const run = async (
     }
     const options = {
       analysis,
+      cwd: cli.cwd,
+      stdin: cli.stdin,
+      console,
     };
     const results = await Promise.allSettled(
       generators.map(async (generator) => {
-        // TODO(kschaaf): Add try/catches around each of these operations and
-        // throw more contextural errors
-        await writeFileTree(out, await generator.generate(options, console));
+        try {
+          await writeFileTree(out, await generator.generate(options));
+        } catch (e) {
+          console.error('Wrapper generation failed', e);
+          return;
```

Comment:
Do we want to return here? That ensures that the error handling below will never run